Gruntfile.js

```diff
@@ -81,15 +81,15 @@ module.exports = function( grunt ) {
       if ( ! _.isEmpty( less_config ) ) {
         grunt.config.set( 'less.' + id, {options: {cleancss: false, compress: true, strictImports: true}, files: less_config} );
         grunt.config.set( 'watch.' + id + '_less', {
-          files: [path.join(basedir, '**/*.less'), path.join(basedir, '**/*.css'), '!' + path.join(basedir, 'vendors/**/*'), '!' + path.join(basedir, '**/*.min.css')],
+          files: [path.join(basedir, '**/*.less'), path.join(basedir, '**/*.css'), '!' + path.join(basedir, 'vendors/**/*'), '!' + path.join(basedir, '**/*.min.css'), '!' + path.join(basedir, 'assets/vendor/**/*')],
```

Comment:
We need these js files in the packaged zip. Let's revert this change to make sure we don't lose these files in builds.

Reply:
This tells grunt not to watch this directory for any changes. The vendor directory should not be included in the path. The same items in the `assets/vendor` directory where in the `vendors`, I have just appended the grunt task to exclude this also

Reply:
You're right. Thanks!